### DISCUSSION TITLE
Adding `gt` tables to PowerPoint presentations has stoped working, how do I get it woking again?

### DISCUSSION THREAD

**Discussion by andresrcs** [quarto-dev/quarto-cli#9363] [Q&A] [answered]
### Description

I used to be able to insert `gt` tables in PowerPoint presentations seamlessly but It has stopped working, here you can find a minimal reproducible example. If you render this code, it produces a presentation with no table in it.

I'm using `quarto` version 1.4.553 that comes bundled with RStudio (daily build 2024.04.0 Build 693)

````qmd
---
title: "Test"
format: pptx
editor: visual
---

```{r setup}
library(gt)
```

## Test Table

```{r}
#|label: test

iris %>% 
    gt() %>%
    as_raw_html()

```
````

**Comment by mcanouil** [comment]
Why are you using `as_raw_html()` for PowerPoint?

**Reply by andresrcs** [reply]
Because that option used to give properly scaled tables

**Reply by cderv** [reply, marked as answer]
@mcanouil we document explicitly that HTML tables can be used to create Tables for non HTML formats
See our doc https://quarto.org/docs/authoring/tables.html#html-tables that says

> Quarto can process HTML tables in html RawBlock nodes (i.e., {=html}) and convert them to Markdown tables, regardless of the output format (intentionally including non-HTML formats). As a result, you can use HTML table syntax in your documents and it will be converted to Markdown syntax for all formats.

That is one of the purpose of Quarto HTML table processing

I confirm this is working for 1.3.450 and not working for 1.4 - so this is a regression where we fail to do what we document. 

It is working for docx or even gfm, so it seems to be related to pptx. 

I'll open an issue with this smaller reprex 

````markdown
---
title: "Tables"
format: 
  pptx: default
keep-md: true
---

# Test table

```{=html}
<table>
<thead>
  <tr>
    <th>Sepal.Length</th>
    <th>Sepal.Width</th>
  </tr>
</thead>
<tbody>
<tr>
  <td>5.1</td>
  <td>3.5</td>
</tr>
<tr>
  <td>4.9</td>
  <td>3.0</td>
</tr>
</tbody>
</table>
```
````

**Reply by mcanouil** [reply]
- https://github.com/quarto-dev/quarto-cli/issues/9365
